### PR TITLE
Add index.html and style.css

### PR DESCRIPTION
**Pull request by riskmaris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,288 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
```
Comment (Qoosim) [inline README.md:65]: - [ ] This is an instruction telling your what you're expected to do. You're expected to replace this placeholder text with the appropriate response. Kindly look into it and fix it too.

File: README.md [hunk 2]
```diff
@@ -1 +1,288 @@
...
+
+**[your_project__name]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
```
Comment (Qoosim) [inline README.md:74]: - [ ] Same thing goes here too. Kindly replace the appropriate response to the placeholder text.

File: README.md [hunk 3]
```diff
@@ -1 +1,288 @@
...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
```
Comment (Qoosim) [inline README.md:105]: - [ ] You're required to give at least one key feature under this heading and remove the placeholder text `Describe between 1-3 key features of the application.`

File: README.md [hunk 4]
```diff
@@ -1 +1,288 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com)
+
```
Comment (Qoosim) [inline README.md:115]: - [ ] It is allowed to write `N/A` under any heading that's not related to your project. But ensure all the headings from the Microverse template present in your README file except `FAQ` which is optional. Kindly fix this.

File: README.md [hunk 5]
```diff
@@ -1 +1,288 @@
...
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (Qoosim) [inline README.md:201]: - [ ] Kindly follow all the instructions under each headings and write `N/A yet` under any one that's not related to your project.

File: README.md [hunk 6]
```diff
@@ -1 +1,288 @@
...
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
Comment (Qoosim) [inline README.md:219]: - [ ] Kindly remove one author and replace all the social media handles with your respective ones.

File: README.md [hunk 7]
```diff
@@ -1 +1,288 @@
...
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
```
Comment (Qoosim) [inline README.md:231]: - [ ] You're required to give at least one future feature under this heading and remove the placeholder text `Describe between 1-3 features of the application.`

File: README.md [hunk 8]
```diff
@@ -1 +1,288 @@
...
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```
Comment (Qoosim) [inline README.md:248]: - [ ] Kindly remove this

File: README.md [hunk 9]
```diff
@@ -1 +1,288 @@
...
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
```
Comment (Qoosim) [inline README.md:258]: - [ ] Kindly remove this too and give your acknoledgements to whoever deserves it.

File: README.md [hunk 10]
```diff
@@ -1 +1,288 @@
...
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
```
Comment (Qoosim) [inline README.md:277]: - [ ] Kindly provide the appropriate questions and answers or remove the `FAQ` completely as it is optional to remain in your README file.

File: README.md [hunk 11]
```diff
@@ -1 +1,288 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Qoosim) [inline README.md:284]: - [ ] The link to the MIT License is broken at this moment. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to set ip up properly for your project.

license

File: README.md [hunk 12]
```diff
@@ -1 +1,288 @@
...
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (Qoosim) [inline README.md:287]: - [ ] Kindly remove this. It is not necessary to be there

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @riskmaris, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @riskmaris  👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight 
✔ Used correct GitFlow👍 
✔ No Linter Errors 👍 

## Required Changes ♻️

- [ ]  Check the comments below 

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — APPROVED** [review]
## Hi @riskmaris 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔

## Optional suggestions
- **[OPTIONAL] As mentioned by the previous reviewer it's not professional to leave the PR description empty as this is where you should provide a well-detailed summary of what you've achieved in this specific milestone 🎯. So please make those improvements in your next project. 👍**
- **[OPTIONAL] Once again As mentioned by the previous reviewer your branch name should be the name of a feature you've added so kindly fix that as well. you can call it for example `basic-structure`**
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**